bookstore/clone.py

```diff
@@ -270,7 +270,7 @@ def build_content_model(self, content, target_path):
         dict
             Jupyter Contents API compatible model
         """
-        path = self.contents_manager.increment_filename(target_path)
+        path = self.contents_manager.increment_filename(target_path, insert='-')
```

Comment:
To bring @chyzzqo2's comment & response over here:

I added this to ensure consistency between the incrementing patterns for s3 and fs cloning.